src/screens/home/modals/last-results-modal.tsx

```diff
@@ -0,0 +1,77 @@
+import { NavigationContainerRef, useNavigation } from '@react-navigation/native';
+import React from 'react';
+import { ScrollView, TouchableOpacity, View } from 'react-native';
+import { Iconify } from 'react-native-iconify';
+
+import { MatchScore } from '~/shared/components/match/match-score';
+import { MatchTeam } from '~/shared/components/match/match-team';
+import { useMatchesResults } from '~/shared/hooks/data/use-matches-results';
+import { useStyles } from '~/shared/hooks/use-styles';
+import { RootStackParamList } from '~/shared/navigation';
+import { createStylesheet } from '~/shared/styles/create-stylesheet';
+
+export const LastResultsModal = React.memo(() => {
+  const styles = useStyles(getStyles);
+
+  const navigation = useNavigation<NavigationContainerRef<RootStackParamList>>();
+
+  const matchs = useMatchesResults();
+
+  if (!matchs.length) {
+    return null;
+  }
```

Comment:
If there is no match, we have to show the back arrow and say to the user that there is no match. If we return null, the user will have no way to go back.

_**same comment for the other modal**_